Kernel/Storage/SD/Registers.h

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Endian.h>
+#include <AK/Types.h>
+
+namespace Kernel::SD {
+
+// SD Host Controller Simplified Specification Version 3.00
+// NOTE: The registers must be 32 bits, because of a quirk in the RPI.
+struct SDRegisters {
```

Comment:
This seems to be missing a size assertion

Reply:
(look for `static_assert(AssertSize<` for many examples)